src/panels/lovelace/cards/hui-entities-card.js

```diff
@@ -122,7 +125,11 @@ class HuiEntitiesCard extends EventsMixin(PolymerElement) {
         !DOMAINS_HIDE_MORE_INFO.includes(computeDomain(entityId))
       ) {
         element.classList.add("state-card-dialog");
-        element.addEventListener("click", () =>
+        longPressBind(element);
+        element.addEventListener("clicked", () =>
```

Comment:
is there a reason we need a custom click event? Because if a click fires, we just don't prevent it if it's within a certain treshold ?

Reply:
The problem is that if a click listener is added, it will run on long press as well, and I can't seem to stop it reliably.

Reply:
It depends a bit on which click listener is bound and evaluated first, the one in LongPress, or the other one.

Reply:
`event.stopPropagation()` should stop events from bubbling further. `event.stopImmediatePropagation();` to stop right away, but that would only work if we are able to make sure we are attached as listener before the click listener, which we can't. 

I guess custom click it is